opentok/opentok-meet
# OPENTOK-46938 add blur options to publisher filters

#### What is this PR doing?
Add background blur with low and high blur strength as options in the publisher filter picker.

#### How should this be manually tested?
- Run locally
- Check that filters work and that all transitions between filters work i.e.
- between blur and none
- between high and low blur
- between blur and existing color filters
- between none and color filters
- blur effects should be cleared when selected other filters (the clear method should be called, you can't tell just by looking at the publisher because the current filters actually create a new publisher, so the blur effect wouldn't be visible anyway).

#### What are the relevant tickets?
https://jira.vonage.com/browse/OPENTOK-46938
